common.yaml.jinja

```
@@ -32,6 +32,7 @@ services:
       traefik.port: "8069"
     {%- endif %}
 
+  {%- if postgres_version is not none %}
```

You can simplify this, here and in all others:

```suggestion
  {%- if postgres_version %}
```